src/looprestoration.rs

```diff
@@ -170,10 +174,22 @@ impl loop_restoration_filter::Fn {
         let dst_ptr = dst.as_mut_ptr::<BD>().cast();
         let dst_stride = dst.stride();
         let left = left[..h as usize].as_ptr().cast();
-        let lpf = lpf.cast();
+        // NOTE: The calculated pointer may point to before the beginning of
```

Comment:
This isn't a safety comment though. None of the operations it's documenting are unsafe, it's just explaining why we need to use `wrapping_offset` instead of `offset`.

Reply:
Yeah, but it's very safety adjacent, and explains why we aren't doing something `unsafe`.  I think it's better to use `SAFETY` for searchability, too.